crates/sui-transactional-test-runner/src/args.rs

```diff
@@ -73,6 +75,12 @@ pub struct SuiInitArgs {
     /// the indexer.
     #[clap(long = "epochs-to-keep")]
     pub epochs_to_keep: Option<u64>,
+    /// TODO (wlmyng): doc comment
+    #[clap(long = "data-ingestion-path")]
```

Comment:
No need to name the long as the field `data_ingestion_path` will be `data-ingestion-path` as an option. I see that the rest of the code is like that, so probably it's better to keep the style the same, but just FYI.